srcpkgs/seexpr-krita/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'seexpr-krita'
+pkgname=seexpr-krita
+version=3.4.4.0
+revision=1
+wrksrc="seexpr-${version}"
+build_style=cmake
+configure_args="-DBUILD_DEMOS=OFF -DBUILD_TESTS=OFF \
```

Comment:
No need to escape a newline in a quoted string. Also don't hard disable tests. add smth like
```
if [ "$XBPS_CHECK_PKGS" ]; then
        configure_args+=" -DBUILD_TESTS=ON"
else
        configure_args+=" -DBUILD_TESTS=OFF"
fi
```
after checksum and

```
do_check() {
        cd build
        # BlockTests requires llvm, we don't build it with llvm
        ctest -E 'BlockTests'
}

post_install() {
        [ -d "${DESTDIR}/usr/share/test/SeExpr2" ] && rm -r "${DESTDIR}/usr/share/test/SeExpr2" || true
}
```
 after pre_configure